spec/features/comparisons_spec.rb

```diff
@@ -60,20 +60,6 @@
       ComparisonService.new(project).refresh_comparisons
     end
 
-    it 'does nothing unless warranted' do
-      project.stages.last.deploy_strategies.create!(
-        provider: 'github pull request',
-        profile: profile,
-        automatic: true,
-        arguments: { base: 'release', head: 'staging ' }
-      )
-      allow_any_instance_of(Releasecop::Checker).to receive(:check).and_return(
-        Releasecop::Result.new('shipping', [small_comparison])
-      )
-      expect(DeployService).not_to receive(:start)
-      ComparisonService.new(project).refresh_comparisons
-    end
-
     it 'does nothing when deploy warranted but automatic is false' do
```

Comment:
Minor but I notice we use the `large_comparison` for the test asserting that it sends -- might make sense to update this to a small comparison or just one change?